src/Agent.Worker/ExecutionContext.cs

```diff
@@ -39,6 +39,7 @@ public interface IExecutionContext : IAgentService
         List<IAsyncCommandContext> AsyncCommands { get; }
         List<string> PrependPath { get; }
         ContainerInfo Container { get; }
```

Comment:
should we rename `Container` to `JobContainer` ? can be done in separate PR if you think that make sense.

Reply:
I would prefer to do this in a separate PR. There are a lot of references that would make this one messy